File: Objectivec/Day6/MapViewDemo/MapViewDemo/DetailsViewController.m
//
//  DetailsViewController.m
//  MapViewDemo
//
//  Created by Esraa Hassan on 3/22/20.
//  Copyright © 2020 Yahya. All rights reserved.
//

#import "DetailsViewController.h"

@interface DetailsViewController ()

@end

@implementation DetailsViewController

- (void)viewDidLoad {
    [super viewDidLoad];
   
    _nameLabel.text = _user.name;
    _phoneLabel.text = _user.phone;
    
    _longitudeLabel.text = _user.longitude;
    _latitudeLabel.text = _user.latitude;
    [_avatarImageView setImage:[UIImage imageNamed:_user.imagePath]];
    
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

@end

File: Swift/Day2/MovieList/MovieList/DetailsViewController.swift
//
//  ViewController.swift
//  MovieList
//
//  Created by Esraa Hassan on 3/25/20.
//  Copyright © 2020 Yahya. All rights reserved.
//

import UIKit

class DetailsViewController: UIViewController {

    @IBOutlet weak var ratingLabel: UILabel!
    @IBOutlet weak var nameLabel: UILabel!
    
    @IBOutlet weak var releaseYearLabel: UILabel!
    
    @IBOutlet weak var genreLabel: UILabel!
    
    @IBOutlet weak var movieImageView: UIImageView!
    
    var movie: Movie?
    
    
    
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        nameLabel.text = movie!.title
        
        movieImageView.image = UIImage.init(named: (movie?.image)!)
        
        
        genreLabel.text = movie!.genre
        
        releaseYearLabel.text = String(movie!.releaseYear)
        ratingLabel.text = String(movie!.rating)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        
    }


}

